jujugui/static/gui/src/app/components/profile/header/header.js

```diff
@@ -0,0 +1,22 @@
+/* Copyright (C) 2017 Canonical Ltd. */
+
+'use strict';
+
+class ProfileHeader extends React.Component {
+
+  render() {
+    return (
+      <div className="profile-header">
+        <div className="content"></div>
```

Comment:
It might be better to BEM your class names here. Something like `profile-header__content`. Reduces the risk of styles leaking.

Then your Sass should be something like this:
``` scss
.profile-header {
  ...

  &__content {
    ...
  }
}
```